<launch>

    <group ns="/airsim_node/Drone1">
        <node pkg="drones_controller" type="drones_controller_node" name="drones_controller_node" output="screen" />
        
        <node pkg="drones_controller" type="blob_detector_node.py" name="drones_controller_node" output="screen" />
    </group>

</launch>